apps/studio/components/interfaces/Settings/Database/UsePoolerCheckbox.tsx

```diff
@@ -66,98 +63,67 @@ export const UsePoolerCheckbox = ({
-                                      className="cursor-pointer w-full text-foreground"
-                                      onSelect={() => {
-                                        onSelectPoolingMode('session')
-                                        setOpen(false)
-                                      }}
-                                      onClick={() => {
-                                        onSelectPoolingMode('session')
-                                        setOpen(false)
-                                      }}
-                                    >
-                                      Session mode
-                                    </CommandItem_Shadcn_>
-                                  </CommandGroup_Shadcn_>
-                                </CommandList_Shadcn_>
-                              </Command_Shadcn_>
-                              <Separator />
-                              <p
-                                className="px-3 py-2 text-xs text-foreground-light hover:text-foreground cursor-pointer transition"
+                          <DropdownMenu>
+                            <DropdownMenuTrigger asChild>
+                              <Button
+                                type="outline"
+                                className="py-0.5 pr-1.5"
+                                iconRight={<IconChevronDown strokeWidth={1.5} />}
+                              >
+                                <span className="capitalize">Mode: {poolingMode}</span>
+                              </Button>
+                            </DropdownMenuTrigger>
+                            <DropdownMenuContent side="bottom" align="start">
+                              <DropdownMenuItem
+                                key="transaction"
+                                className="text-foreground"
+                                onClick={() => onSelectPoolingMode('transaction')}
+                              >
+                                <Tooltip.Root delayDuration={0}>
+                                  <Tooltip.Trigger asChild>
+                                    <span className="w-full">Transaction mode</span>
+                                  </Tooltip.Trigger>
+                                  {data.pool_mode === 'session' && (
+                                    <Tooltip.Portal>
+                                      <Tooltip.Content side="right">
+                                        <Tooltip.Arrow className="radix-tooltip-arrow" />
+                                        <div
+                                          className={[
+                                            'rounded bg-alternative py-1 px-2 leading-none shadow',
+                                            'border border-background w-[380px]',
+                                          ].join(' ')}
+                                        >
+                                          <span className="text-xs text-foreground">
```

Comment:
text-light

Reply:
let's do this when we move the tooltip to the UI library - all the tooltips across the dashboard is using text-foreground